bors.toml

```diff
@@ -1,6 +1,7 @@
 # Test suite must pass on both Travis and AppVeyor before merging into `master`.
 status = [
-    "continuous-integration/travis-ci/push"
+    "continuous-integration/travis-ci/push",
+    'ci/gitlab/%',
```

Comment:
:zzz: next time we try to merge anything.

Reply:
Fixed

Reply:
It completes in about 30 minutes on GitLab.